crates/bevy_render/src/camera/camera.rs

```diff
@@ -14,16 +14,65 @@ use bevy_transform::components::GlobalTransform;
 use bevy_window::{WindowCreated, WindowId, WindowResized, Windows};
 use serde::{Deserialize, Serialize};
 
+/// Camera component defining what to render.
+///
+/// The camera component is added to a world to define the properties of the viewpoint from
+/// which rendering occurs. It defines the position of the view to render, the projection method
+/// to transform the 3D objects into a 2D image, as well as the window into which that image
+/// is drawn.
+///
+/// Adding a camera is typically done by adding a bundle, either the [`OrthographicCameraBundle`]
+/// or the [`PerspectiveCameraBundle`].
+///
+/// The internal Bevy renderer makes a distinction between 2D and 3D content, which are rendered
+/// separately. To that end, at most two cameras should be added to the world, one per content.
+/// The content type is determined by the camera name, which is automatically set as appropriate
+/// by the various bundles.
+///
+/// [`OrthographicCameraBundle`]: crate::camera::OrthographicCameraBundle
+/// [`PerspectiveCameraBundle`]: crate::camera::PerspectiveCameraBundle
 #[derive(Component, Default, Debug, Reflect)]
 #[reflect(Component)]
 pub struct Camera {
+    /// The projection matrix of the camera, transforming objects from the 3D camera space to the
+    /// 2D screen space.
+    ///
+    /// If a [`CameraProjection`] component is present on the same entity as this camera, this
+    /// projection matrix is automatically updated each frame based on that [`CameraProjection`]
+    /// component.
     pub projection_matrix: Mat4,
+    /// The camera name. This is used to identify the active 2D and 3D cameras, by assigning
+    /// the [`CameraPlugin::CAMERA_2D`] and [`CameraPlugin::CAMERA_3D`] predefined names,
+    /// respectively. See [`ActiveCameras`] for details.
+    ///
+    /// [`CameraPlugin::CAMERA_2D`]: super::CameraPlugin::CAMERA_2D
+    /// [`CameraPlugin::CAMERA_3D`]: super::CameraPlugin::CAMERA_3D
+    /// [`ActiveCameras`]: crate::camera::ActiveCameras
     pub name: Option<String>,
+    /// The window into which this camera rendering is drawn.
     #[reflect(ignore)]
     pub window: WindowId,
+    /// The depth calculation method used when rendering with this camera.
     #[reflect(ignore)]
     pub depth_calculation: DepthCalculation,
+    /// Distance of the near clipping plane from the camera position.
+    ///
+    /// All fragments, produced by rendering the 3D objects, which are closer than this distance
```

Comment:
Do we have a link to what a "fragment" is, particularly in the context of bevy's renderer?

Reply:
I was going for the terminology of the hardware; no idea if there's something in the new renderer about this.

Reply:
Outdated; this was removed with latest merge of main.